air-script/tests/helpers.rs

```diff
@@ -1,4 +1,5 @@
-use codegen_winter::CodeGenerator;
+use codegen_gce::GCECodeGenerator;
+use codegen_winter::WinterfellCodeGenerator;
```

Comment:
Once we have both of these in the `air_script` library, you can import as:

```
use air_script::{GceCodeGenerator, WinterfellCodeGenerator};
```

I think we should actually do the same with the IR and parser inputs, so we would have:


```
use air_script::{parse, AirIR, GceCodeGenerator, WinterfellCodeGenerator};
```